yt_dlp/extractor/digiview.py

```diff
@@ -0,0 +1,51 @@
+from .youtube import YoutubeIE
+from ..utils import clean_html, int_or_none, traverse_obj, url_or_none, urlencode_postdata
+
+
+class DigiviewIE(YoutubeIE):
```

Comment:
Don't subclass from a concrete extractor
```suggestion
class DigiviewIE(InfoExtractor):
```
Needs `from .common import InfoExtractor` as well